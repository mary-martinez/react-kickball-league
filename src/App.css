ul {
  list-style: none;
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  background-color: gray;
}
li {
  font-size: 1.5rem;
  margin: 10px;
  text-decoration: none;
  border-radius: 5px;
}

.App {
  text-align: center;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
.page-content {
  margin-top: 10%;
}
.active {
  background-color: white;
  text-decoration: none;

}
.loader {
  font-size:48px;
  margin-left: 48%;
  margin-top: 30%;
  color: rgb(53, 166, 194);
  width: 1em;
  height: 1em;
  box-sizing: border-box;
  background-color: currentcolor;
  position: relative;
  border-radius: 50%;
  transform: rotateX(-60deg) perspective(1000px);
}
.loader:before,
.loader:after {
  content: '';
  display: block;
  position: absolute;
  box-sizing: border-box;
  top: 0;
  left: 0;
  width: inherit;
  height: inherit;
  border-radius: inherit;
  animation: flowerFlow 1s ease-out infinite;
}
.loader:after {
  animation-delay: .4s;
}

@keyframes flowerFlow {
  0% {
    opacity: 1;
    transform: rotate(0deg);
    box-shadow: 0 0 0 -.5em currentcolor,
    0 0 0 -.5em currentcolor,
    0 0 0 -.5em currentcolor,
    0 0 0 -.5em currentcolor,
    0 0 0 -.5em currentcolor,
    0 0 0 -.5em currentcolor,
    0 0 0 -.5em currentcolor,
    0 0 0 -.5em currentcolor;
  }
  100% {
    opacity: 0;
    transform: rotate(180deg);
    box-shadow: -1em -1em 0 -.35em currentcolor,
    0 -1.5em 0 -.35em currentcolor,
    1em -1em 0 -.35em currentcolor,
    -1.5em 0 0 -.35em currentcolor,
    1.5em -0 0 -.35em currentcolor,
    -1em 1em 0 -.35em currentcolor,
    0 1.5em 0 -.35em currentcolor,
    1em 1em 0 -.35em currentcolor;
  }
}
