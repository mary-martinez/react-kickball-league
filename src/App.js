import { BrowserRouter, Route, Switch } from 'react-router-dom';
import './App.css';
import Header from './components/Header/Header';
import Home from './views/Home/Home';
import Teams from './views/Teams/Teams';

function App() {
  return (
    <BrowserRouter>
      <Header />
      <div className="App">
        <Switch>
          <Route exact path='/'>
            <Home />
          </Route>
          <Route path='/teams'>
            <Teams />
          </Route>
        </Switch>
      </div>
    </BrowserRouter>
  );
}

export default App;
