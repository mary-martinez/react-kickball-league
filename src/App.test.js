import { render, screen } from '@testing-library/react';
import App from './App';

test('renders Home link', () => {
  const app = render(<App />);
  const linkElement = screen.getByText(/Home/i);
  expect(linkElement).toBeInTheDocument();
  expect(app).toMatchSnapshot();
});
