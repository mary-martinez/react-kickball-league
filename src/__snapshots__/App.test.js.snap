// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home link 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <ul>
          <li>
            <a
              aria-current="page"
              class="active"
              href="/"
              style="text-decoration: none;"
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/teams"
              style="text-decoration: none;"
            >
              Teams
            </a>
          </li>
          <li>
            <a
              href="/players"
              style="text-decoration: none;"
            >
              Players
            </a>
          </li>
        </ul>
      </div>
      <div
        class="App"
      >
        <div>
          <h1
            class="headings"
          >
            Welcome to the PNW Kickball League!
          </h1>
          <div
            class="page-content"
          >
            <h3>
              Check out our 
              <a
                href="/teams"
              >
                local teams
              </a>
            </h3>
            <h3>
              Or check in on your favorite 
              <a
                href="/players"
              >
                players
              </a>
            </h3>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <ul>
        <li>
          <a
            aria-current="page"
            class="active"
            href="/"
            style="text-decoration: none;"
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/teams"
            style="text-decoration: none;"
          >
            Teams
          </a>
        </li>
        <li>
          <a
            href="/players"
            style="text-decoration: none;"
          >
            Players
          </a>
        </li>
      </ul>
    </div>
    <div
      class="App"
    >
      <div>
        <h1
          class="headings"
        >
          Welcome to the PNW Kickball League!
        </h1>
        <div
          class="page-content"
        >
          <h3>
            Check out our 
            <a
              href="/teams"
            >
              local teams
            </a>
          </h3>
          <h3>
            Or check in on your favorite 
            <a
              href="/players"
            >
              players
            </a>
          </h3>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
